### PR TITLE
Fix doc of op datastore

### PR DESCRIPTION
**Pull request by AurelienFT**

### CODE REVIEW

**Comment by AurelienFT** [comment]
I don't know why it has been closed @qdrn but it shouldn't it's not related to warnings

**Comment by qdrn** [comment]
> I don't know why it has been closed @qdrn but it shouldn't it's not related to warnings

I think #139 was wrongly linked to this issue.

**Review by qdrn — COMMENTED** [review]
I don't understand this part:
`with the functions in this namespace`

**Comment by AurelienFT** [comment]
> I don't understand this part: `with the functions in this namespace`

You can use the functions in the `OpDatastore` space to play with the op datastore read etc

File: docs/build/smart-contract/as-sdk.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The Contract namespace includes functions to interact with other smart contract
 
 ## OpDatastore
 
-The OpDatastore namespace includes functions to interact with the operation datastore. This namespace is particularly useful to read and write data that is expected to persist between contract executions, such as contract state or user information. For more information on this namespace, see the [AS-SDK OpDatastore documentation](https://as-sdk.docs.massa.net/modules/OpDatastore.html).
+The OpDatastore namespace includes functions to interact with the operation datastore. This namespace is particularly useful to read data that is passed with the bytecode in a execute smart contract operation. This key/value storage can be filled by the operation creator and is readable by the smart contract executed with the functions in this namespace. For more information on this namespace, see the [AS-SDK OpDatastore documentation](https://as-sdk.docs.massa.net/modules/OpDatastore.html).
```
Comment (qdrn) [inline docs/build/smart-contract/as-sdk.md:32]:
```suggestion
The OpDatastore namespace includes functions to interact with the operation datastore. This key/value storage can be filled by the operation creator and then used by the smart contract being executed. This namespace is particularly useful to read the data that is passed with the bytecode in a execute smart contract operation. For more information on this namespace, see the [AS-SDK OpDatastore documentation](https://as-sdk.docs.massa.net/modules/OpDatastore.html).
```

**Review by qdrn — COMMENTED** [review]
I've suggested a modification based on my understanding. WDYT ?

**Comment by AurelienFT** [comment]
Yes it's pretty close to what I said I accept

**Comment by qdrn** [comment]
> Yes it's pretty close to what I said I accept

Thanks good sir.

**Review by qdrn — APPROVED** [review]

**Pull request merged**